events/importer/harrastushaku.py

```diff
@@ -73,16 +73,17 @@ def setup(self):
     def import_places(self):
         """Import Harrastushaku locations as Places
 
-        - If a place in Harrastushaku has "tprek_id" set, assume it is a place that
-          comes from Toimipisterekisteri and don't import it from Harrastushaku.
+        - If we can find a close-enough match for the location object coming from Harrastushaku in Toimipisterekisteri,
+          we do not import that location object, as this this will cause duplicate location issue due to
+          Harrastushaku data being of low quality.
 
-        - If a place in Harrastushaku does not have "tprek_id", it will be
-          imported having data source "harrastushaku".
+        - If, however, we cannot find a match, location object will be imported with data source "harrastushaku".
         """
         logger.info('Importing places...')
 
         locations = self.fetch_locations()
         logger.debug('Handling {} locations...'.format(len(locations)))
+        self.mapped_harrastushaku_locations = self.map_harrastushaku_location_ids_to_tprek_ids(locations)
```

Comment:
Hmm, is there a reason why this  cannot be named `self.location_id_to_place_id`, as it seems to be exactly the same thing?

Reply:
Correct. Done in 935714dac1615e46a1f88e62f58385df92124ded